lib/heimdall_tools/sonarqube_mapper.rb

```diff
@@ -237,7 +237,7 @@ def get_nist_tags
       return [@mappings[tag_type][parsed_tag]].flatten.uniq
```

Comment:
If this fuction is only used by the `sonarqube` module we should identify it as `get_nist_tag_for_sonarqube` to make the code more clear and contained.

Reply:
If this is a reused by other elments, then we have to make sure that we can support the fact that for `sonarqube` SA-11 and RA-5 are for SONARQUBE and other controls are the default for other modules.

Reply:
@aaronlippold has this comment been addressed?